js/privatebin.js

```diff
@@ -602,9 +602,9 @@ jQuery.PrivateBin = (function($) {
         me.formatBytes = function (bytes)
         {
             let result = '';
-            const kilobyte = 1024;
+            const kilobyte = 1000;
             const decimalPoint = 2;
-            const sizes = [I18n._('B'), I18n._('KiB'), I18n._('MiB'), I18n._('GiB')];
+            const sizes = [I18n._('B'), I18n._('kB'), I18n._('MB'), I18n._('GB')];
```

Comment:
The following is questionable and unlikely to matter in the foreseeable future... just to match the function in PHP as well as the translated strings, we could:
```suggestion
            const sizes = [I18n._('B'), I18n._('kB'), I18n._('MB'), I18n._('GB'), I18n._('TB'), I18n._('PB'), I18n._('EB'), I18n._('ZB'), I18n._('YB')];
```

Reply:
I don't mind this change, we just have to regenerate SRI for privatebin.js instanse